### ISSUE TITLE
Renew Apple Mac Certificates

### ISSUE THREAD

**Issue by rat-moonshine** [prominic/Moonshine-IDE#678]
![image](https://user-images.githubusercontent.com/24370870/76086160-8d37ae80-5fd9-11ea-9037-3de6c725f359.png)

- [x] Renew all relevant certificates
- [x] Ensure installed to build systems

**Comment by rat-moonshine** [comment]
I was having some problem compiling projects lately on my local build system, this possibly introduced since upgrade to macOS Mojave. Although, the certificates are still valid and non-expired in my system's keychain. Looking through the build errors, it seems the process having problem building with certificates from keychain. This may needs some more investigation at my end.

Since the above errors, I thought to follow a process from past and install/export newer/renewed certificates. The process involves generating new Apple certificates following certificate-signing-request from my local build system's keychain, install to my system's keychain, if everything goes Okay next to export the certificates from my keychain and install to Bamboo build-system's keychain. Since the certificates generated by my system's signing-request, a direct install to Bamboo build-system's keychain may not work.

I generated two required Mac certificates, however I continued to get same errors during local build. I revoked those certificates and re-do the process from opposite direction:

1. Generated certificate-signing-request from Bamboo build system
2. Generated two new Mac certificates using the same 
3. Installed to Bamboo build system's keychain
4. Ensured build is working correctly using newly installed certificates 

I confirm builds are working properly - https://bamboo.prominic.net/browse/MS-MGOSX-1616 is compiled with newly installed certificates. 

Next step, to install the requisite certificates in my build-system exported from Bamboo build-system. I shall look into this, later.
